Lib/python/pyrun.swg

```diff
@@ -1581,7 +1628,7 @@ SWIG_Python_NonDynamicSetAttr(PyObject *obj, PyObject *name, PyObject *value) {
     } else {
       encoded_name = PyUnicode_AsUTF8String(name);
       if (!encoded_name)
-        return -1;
+        goto done;
```

Comment:
There is still a reference on `name`. This is cleaned up by the code at `done`, and `res` is still at its default value of `-1`:
```c
  done:
  Py_DECREF(name);
  return res;
```